parser_email.py

```diff
@@ -884,11 +903,12 @@ def _handle_mail_object(mail, email_id, rfc822_email, tmp_dir, start_time_epoch)
         _parse_email_headers(parsed_mail, mail, add_email_id=email_id)
         # parsed_mail[PROC_EMAIL_JSON_EMAIL_HEADERS].append(mail.items())
         file_path = "{0}/part_1.text".format(tmp_dir)
+        file_name = mail.get_filename() or mail.get('Subject', 'part_1.txt')
```

Comment:
Similar to how we manually set the file_path, we can set a default value for this one as well.
This is safe, as we're dealing with a single part email.

Reply:
@achandya-crest Thank you for your suggestion on this part. Could you check if this is something you have in mind?

Reply:
this one is `part_1.txt` while the file_path has `part_1.text`, is that intentional? also can we make these default names constants so they're easier to find?

Reply:
Great suggestions! updated.